src/components/ParameterChangeForm.tsx

```diff
@@ -1,21 +1,231 @@
-import React, { FormEvent } from "react";
+import React, { FormEvent, useEffect, useRef, useState } from "react";
 import { Button } from "./Button";
+import { useNetwork } from "../hooks/useNetwork";
+import { ParamChange } from "cosmjs-types/cosmos/params/v1beta1/params";
+import { ProposalArgs } from "./ProposalForm.tsx";
 
 interface ParameterChangeFormProps {
   title: string;
   description: string;
+  handleSubmit: (proposal: ProposalArgs) => void;
 }
 
+type StringBeans = { key: string; beans: string };
+type PowerFlagFee = { power_flag: string; fee: Coins[] };
+type Coins = { denom: string; amount: string };
+/**
+ * TODO: import from "@agoric/cosmic-proto/swingset/swingset.js"
+ * which uses camelCase
+ */
+type SwingsetParams = {
+  beans_per_unit: StringBeans[];
+  fee_unit_price: Coins[];
+  bootstrap_vat_config: string;
+  power_flag_fees: PowerFlagFee[];
+};
```

Comment:
these might need to be camel case depending how it's encoding. (We send in camel and they come out snake)